framework/src/main/java/org/checkerframework/framework/util/dependenttypes/DependentTypesHelper.java

```diff
@@ -975,11 +990,13 @@ public void checkMethod(MethodTree methodDeclTree, AnnotatedExecutableType type)
      * @param methodType annotated type of the method
      */
     private void checkTypeVariables(MethodTree node, AnnotatedExecutableType methodType) {
-        JavaExpressionContext context =
-                JavaExpressionContext.buildContextForMethodDeclaration(node, factory.getChecker());
         for (int i = 0; i < methodType.getTypeVariables().size(); i++) {
             AnnotatedTypeMirror atm = methodType.getTypeVariables().get(i);
-            viewpointAdaptToContext(context, atm);
+            convertAnnotatedTypeMirror(
+                    stringExpr ->
+                            StringToJavaExpression.atMethodBody(
+                                    stringExpr, node, factory.getChecker()),
+                    atm);
             checkType(atm, node.getTypeParameters().get(i));
         }
     }
```

Comment:
`isExpressionAnno`'s documentation says "whose value" but I think it means, "if any element's declared type".

Also, can that `for` loop in it be made more efficient?